src/pages/index.js

```diff
@@ -22,19 +22,24 @@ const categories = [{
 }]
 
 // e.g. normalise `fruit-veg` -> `Fruit & Veg` with react-i18next or other i18n lib
-const normaliseTag = tag => tag;
+// const normaliseTag = tag => tag;
 
 const extractItemData = ({
   node: {
     Name = '',
     Category_1 = '',
     Tags,
+    Collection,
+    Delivery,
+    Image_Url,
   } = {},
 }) => ({
   name: Name,
   category: Category_1,
-  source: 'https://media-cdn.tripadvisor.com/media/photo-s/15/7d/ca/8a/bakery.jpg',
-  tags: normaliseTag(Tags) || ['Fruit & Veg', 'Bread'],
+  source: Image_Url,
+  tags: Tags.split(","),
```

Comment:
This should be a pure function I think, e.g. `normaliseTag` would do `tags.split`, etc